backend/typescript/types.ts

```diff
@@ -16,6 +16,15 @@ export type UserDTO = {
   role: Role;
 };
 
+export type DonorDTO = {
+  donorId: number;
```

Comment:
shouldnt this be just `id` instead of `donorID`. Also do you know why the DTOs have camelCase fields and models have snake_case?

Reply:
I think we should rename this as `donorID`